src/Adapter/Cart/CommandHandler/AddCustomizationFieldsHandler.php

```diff
@@ -110,4 +116,73 @@ public function handle(AddCustomizationFieldsCommand $command): int
 
         return $customizationId;
     }
+
+    /**
+     * Uploads image from customized field
+     *
+     * @param UploadedFile $file
+     *
+     * @return string
+     *
+     * @throws FileUploadException
+     */
+    private function uploadImage(UploadedFile $file): string
+    {
+        $this->validateUpload($file);
+
+        //@todo: check if copy is okay to use instead of move_uploaded_file(this fails creating new request from global later)
```

Comment:
According to Stackoverflow (a very accurate source of data 😄), we can use `Symfony\Component\HttpFoundation\File\UploadedFile::move()` instead

Reply:
I've tried, but it requires directory and a file name to be separate, so it doesn't go well with the generated tmpName as it contains both already, so I'm not really sure how to workaround this without ruining something ? :smile: